README.md

```diff
@@ -22,10 +22,17 @@ Checking for node:
     npm install
 
 
-SQLite3 may need to be built from source in order to comply with the version of Node on your system, as well as the system's architecture. If running `node app.js` or `nodejs app.js` shows errors with SQLite, run the following:
+SQLite3 may need to be built from source in order to comply with the version of Node on your system, as well as the system's architecture. Ensure the apt package for sqlite3 is downloaded:
+
+    sudo apt install sqlite3
```

Comment:
This has a lot of overlap with `SQLite3` already being listed as a prerequisite; could the prerequisites section instead be clarified? (Also, it's possible that the system Nabu is being installed on won't be Linux - at least one of our colleagues develops on a Mac)

Reply:
Yeah that makes sense. Do you think changing the requirements to something like: _SQLite3 (system software, npm module will be installed later)_ would be ok instead? That's what confused me at least

Reply:
Yes, could compact that to `SQLite3 (system software)` and what do you think about modifing the start of line 25 to be:
`The npm SQLite3 module may need to be built from source in order to comply with...`